htdocs/societe/card.php

```diff
@@ -1728,7 +1736,11 @@ function formatCustomerSelection (selection) {
 				print "\n";
 				print '<script type="text/javascript">';
 				print "function CheckVAT(a) {\n";
-				print "newpopup('".DOL_URL_ROOT."/societe/checkvat/checkVatPopup.php?vatNumber='+a, '".dol_escape_js($langs->trans("VATIntraCheckableOnEUSite"))."', ".$widthpopup.", ".$heightpopup.");\n";
+				if ($mysoc->country_code == 'GR' && $object->country_code == 'GR') {
```

Comment:
Can you add a test on   !empty($u)
so the GRVAT is used only for GR AND if setup of web service is defined.
Itfallback on common webservice for all european countries it not.

Reply:
test added